smbios.adoc

```diff
@@ -16,24 +16,23 @@ NOTE: The structures and fields in this section are defined in a manner consiste
 [%header, cols="5,25"]
 |===
 | ID#     ^| Requirement
-| `SMBIOS_010` | A Baseboard/Module Information (Type 02) structure SHOULD be implemented.
-| `SMBIOS_020` | A System Enclosure/Chassis (Type 03) structure SHOULD be implemented.
+| `SMBIOS_010` | The essential structures and data from Table 4, Section 6.2 of the cite:[SMBIOS] specification, MUST be implemented if hardware support is present.
```

Comment:
You're basically saying here that SMBIOS spec requirements should be honored. When working on ACPI and UEFI sections, this was a particular pain point. Originally we had followed the model set by SystemReady specs, which would duplicate requirements. Many on the BRS team felt this was confusing and unnecessary.

It should not be necessary to state this requirement. We already have "This section defines the BRS-I mandatory and optional SMBIOS requirements on top of existing cite:[SMBIOS] specification requirements." at the top.

Reply:
Okay, point taken. I will put back the original statement of the SMBIOS_010.